### ISSUE TITLE
Decide on version numbering, use git tags

### ISSUE THREAD

**Issue by AlexDaniel** [CBPP/cbpp#23]
What would be the version format? Some clarification is needed.
One of the most popular formats in software development is [Semantic Versioning](http://semver.org/) (or other custom formats that look almost the same).
However, your previous torrent files had a date in them. Indeed, different distros handle this question in a different way. It would be great to put a bit more thought into it.

Also, I believe that every release must be tagged in git. If you're not familiar with git tags, please read this: http://git-scm.com/book/en/v2/Git-Basics-Tagging.
Basically it means that every tag will appear [on this page](https://github.com/CBPP/cbpp/releases) where you can write a meaningful description and attach any number of files (torrent files belong here, I think).

**Comment by AlexDaniel** [comment]
This issue is related to https://github.com/CBPP/cbpp/issues/16, but here the question is "Should we even use dates?".

**Comment by computermouth** [comment]
Adopting Semantic seems like a good option. So far I'd just been calling all our new ports of the packages 1.0, but adding in an extra .0 wouldn't be an issue. The image/torrent were named that way on a whim. But this looks like a good place to start.

**Comment by amharris** [comment]
@computermouth Linus Torvalds recently opted not to go for the longer numbers for versioning.
...but, as with most things, there tends to at least be a degree of subjectivity involved -- so it depends, you're heading this: what do _you_ want?

I'd personally rather not get too lengthy with the version numbers, otherwise it can be tricky to keep a track of things.

I have a multitude of varying views when it comes to version numbering but do like what's done with Node.js

**Comment by computermouth** [comment]
All patches applied to packages were released under a "1.0" version number. Subsequent patches will result in point releases per commit.
